lib/client/src/utils/ArrayUtils.js

```diff
@@ -16,31 +17,32 @@
    * @param replace Number of elements to remove
    * @param arrayToSplice Optional array to append
    */
-  var spliceArray = function(origArray, start, replace, arrayToSplice) {
+  wilson.utils.spliceArray = function spliceArray(origArray, start, replace, arrayToSplice) {
     var args = [start, replace];
     if (arguments.length > 3) { args = args.concat(arrayToSplice); } // In case arrayToSplice is not passed in, otherwise appending 'undefined'
     Array.prototype.splice.apply(origArray, args);
   };
 
+
   /**
    * Replaces the contents of @origArray with the contents of @newArray
+   *
    * @param origArray
    * @param newArray
    */
-  var replaceArray = function(origArray, newArray) {
-    spliceArray(origArray, 0, origArray.length, newArray);
+  wilson.utils.replaceArray = function replaceArray(origArray, newArray) {
+    wilson.utils.spliceArray(origArray, 0, origArray.length, newArray);
```

Comment:
@easyspeak I added a var in front: 

`var spliceArray = wilson.utils.spliceArray = function spliceArray()....`